packages/eslint-plugin/src/rules/no-implied-eval.ts

```diff
@@ -1,3 +1,4 @@
+import { isFunctionLike } from '@typescript-eslint/type-utils';
```

Comment:
`@typescript-eslint/type-utils` shouldn't be imported directly. This function should probably be imported from `src/utils`, as other rules do

https://github.com/typescript-eslint/typescript-eslint/blob/920f909debc3eb394f5aef4e36568bdf6641e304/packages/eslint-plugin/src/util/index.ts#L18-L19